fw/vhost.c

```diff
@@ -1904,6 +1908,19 @@ __tfw_cfgop_frang_http_methods(TfwCfgSpec *cs, TfwCfgEntry *ce,
 		methods_mask |= (1UL << method_id);
 	}
 
+	if (!ce->val_n) {		
+		if (!ce->dflt_value) {
+			T_ERR_NL("frang: Empty http_methods.");
+			return -EINVAL;
+		}
+
+		T_ERR_NL("frang: http_methods should contain at least one "
```

Comment:
Looks like dead code and we never reach this place. If we have `ce->dflt_value == false` it means value was specified in config and now we know that this value is empty. Therefore, we will always return in `if (!ce->dflt_value)`. Is there a case when `ce->dflt_value == true` but `ce->val_n == false`?

Reply:
This situation is possible  when we have  " http_methods  ;" in config

Reply:
In my understanding  when we have " http_methods ;" then we have `ce->dflt_value == false` and `ce->val_n == false` in other cases we will have `ce->val_n > 0`. For example: http_methods ""; or http_methods " "; will have `ce->val_n > 0` and will disallowed in `TFW_CFG_ENTRY_FOR_EACH_VAL`. I can't find another value for http_methods where `ce->val_n == 0` except " http_methods ;". Do you know such value for http_methods?

Reply:
Yes, when we have 
		.name = "http_methods",
		.deflt = "",
in the code

Reply:
Now we doesn't, and I believe we must not maintain this. If you warring about this, it would be better to write a comment in appropriate place.